controllers/acquire_cube.py

```diff
@@ -57,6 +64,12 @@ def grabbing(self) -> None:
         self.gripper.close(GamePiece.CUBE)
         if self.gripper.get_full_closed():
             self.status_lights.cube_onboard()
+            self.next_state("raising")
+
+    @state(must_finish=True)
+    def raising(self):
+        self.arm.go_to_setpoint(Setpoints.HANDOFF, do_retract=False)
+        if self.arm.at_goal():
```

Comment:
I don't really understand what adding this will do. It will already be at the handoff setpoint. Wont issuing this again with ```do_retract=False``` mean that nothing happens or have I assumed the incorrect functionality in ```arm.go_to_setpoint```. 

TLDR: I think this whole hunk can probably go because it does nothing

Reply:
If you let the recover controller move the arm, it will try to retract first which pulls the cube out of the end effector when it gets stuck on the new backboard of the well.

Reply:
But isn't that going to happen anyway because it's already in the handoff position by the time it begins this state and its told to go there again? It will just remain static and won't rotate will it? if you want to get past the intake without hitting the backboard then it sounds to me like you want the following:
```suggestion
            self.next_state("raising")

    @state(must_finish=True)
    def raising(self):
        self.arm.go_to_setpoint(Setpoints.STOW, do_retract=False)
        if self.arm.at_goal():
```

This way it will rotate outside of the collision zone and then once the recover state machine engages, it will effectively just retract

Reply:
But isn't that going to happen anyway because it's already in the handoff position by the time it begins this state and its told to go there again? It will just remain static and won't rotate will it? if you want to get past the intake without hitting the backboard then it sounds to me like you want the following:
```suggestion
            self.next_state("raising")

    @state(must_finish=True)
    def raising(self):
        self.arm.go_to_setpoint(Setpoints.STOW, do_retract=False)
        if self.arm.at_goal():
```

This way it will rotate outside of the collision zone and then once the recover state machine engages, it will effectively just retract

Reply:
Yes, I think that was intended to go to stow